scripts/check-pkg-for-release.js

```diff
@@ -212,7 +212,8 @@ function checkComponentImportedAddedInMDCPackage(ast) {
     'ImportDeclaration'({node}) {
       if (node.source) {
         const source = node.source.value;
-        if (source === pkg.name + '/index') {
+        const pkgFile = pkg.name + '/index';
+        if (source === pkgFile || source === pkgFile + '.ts') {
```

Comment:
Should we be able to re-simplify this again once everything's converted to TS? If so I'd add a TODO comment here (and anywhere else you can think of, e.g. where we're currently accounting for JS + TS).

Reply:
ya good idea...want me to also create an issue?

Reply:
I added a couple of post-conversion items to #4225.